### DISCUSSION TITLE
A way to log request body?

### DISCUSSION THREAD

**Discussion by uap-dev** [ory/hydra#3352] [Q&A]
Is there a way to log the request body not including any sensitive fields? I noticed that the logging does not include any of the request body information, which is currently making it a challenge to understand specific failures. Ty!

**Comment by vinckr** [comment]
Hello @TroutZen 
Sorry for the late answer! 

In any case, if you want to include sensitive values in the logs, you can set the configuration key `log.leak_sensitive_values` to `true` or set the environment variable `LOG_LEAK_SENSITIVE_VALUES` to `true`. 
Be aware that this could potentially expose sensitive data in your logs.
The request body is not included in the logs. You might need to look into error handling and reporting mechanisms specific to the failures you are encountering.

For more information, you can refer to the [Ory documentation on logging](https://www.ory.sh/docs/self-hosted/operations/logging).
